### ISSUE TITLE
Framing

### ISSUE THREAD

**Issue by senhongo** [senhongo/shed#5]
- [x] measure floor panel spacing
- [x] cut floor panel spacing
- [x] side wall (R)
- [x] side wall (L)
- [x] back wall
- [x] front wall
- [x] roof
- [x] panels

**Comment by senhongo** [comment]
Did side wall R today. Rough estimate, took slightly more than 30 minutes.

### Lessons

- pre-drill all the holes at once
- pre-set all the screws at once

**Comment by senhongo** [comment]
Was thinking about how to add the wall panels.
Problem was I wasn't sure how to get the roof angle.

I think the solution is that, we don't bother with the roof angle.

We apply the panels onto the current wall.

Then when we get around to the triangular bit, we'll fill those spots up using off cuts.

**Comment by senhongo** [comment]
Also, let's make life easier for ourselves.

Fill in the floor panels first, then assemble the framing on top of the floor.

That means we'll have to measure, and the the floor panels to measure.

**Comment by senhongo** [comment]
Added additional bracing for the floor panels, mounted boards onto the right wall and also put it up today. Was preparing to pre-drill for the L wall but my drill fried it self.

**Comment by senhongo** [comment]
Framed the Left wall today. Did not mount panels as I wasn't feeling well and didn't want to push myself.

The plan now is 
- monday: panels and put up
- tuesday: frame back wall
- wednesday: back wall panels and put up
- thursday: frame front wall
- friday: front wall panels and put up
- saturday: roof framing

Probably won't get to do all this as I'm still not feeling well and will likely need to take a break tomorrow. Also need to focus on Chubic.

**Comment by senhongo** [comment]
This plan has failed because I got covid.

We need a new plan.

**Comment by senhongo** [comment]
We have a plan.

### Progress
- I mowed the overgrown grass and cleared the spider webs
- I added additional bracing to some of the scaffolding and added a walkway
- the walkway feels a bit soft now, I need to add additional bracing to it

### The plan
Not really a plan. Just do as much as I can this weekend.
Hope to finish all four walls this weekend, so I can finally start working on the roof.

TODO moved to top comment

**Comment by senhongo** [comment]
Almost had an accidental today. Actually I had an accident, I dropped the wall. I just happened to be standing in the window area and was unhurt.

Today's lesson is that shit is heavy and I am still not fully recovered from covid, particularly strength.

For the north wall, rather than one giant block, I shall split it in two.

**Comment by senhongo** [comment]
The wall panels should also be split into upper and lower halves. The upper half can be pre-mounted, we'll add the lower once once we've stood up the wall.

**Comment by senhongo** [comment]
Was physically exhausted yesterday and slept late into the afternoon, so I only had about 2~3 hours available to work.

I split the back wall into 2 separate parts left and right. I've also split the wall panels into upper and lower halves. Right now, the left framing is built and up, and the upper panels have been mounted. I will complete the right half of the wall tomorrow.

As for the lower panels, I will probably put off working on them until the roof is up, just so that there is airflow and moisture can escape.

**Comment by senhongo** [comment]
I've assembled the framing of the right part of the north wall, and also cut the wall panels.

I decided not to mount them up or stand it up today, because I need to open a new bottle of caulk and I'd rather use it together with the front wall. So in the coming few days, I'll assemble the front wall as well, then stand both up together at the same time.

Also, I've noticed that some of the 2x4 for  the front wall is almost rotting and severely warped. Because they were placed under a part of the roof that catches the condensation and keeps dripping all night long. I should've placed those back in the garage when I realise I needed to take a couple of months off. But what's done is done.

I'll replace them with fresh 2x4s and reuse the top and bottom plates for scaffolding or something.

I should be able to start working on the roof this weekend so I'll give a call to Yokoyama to get the windows and roofing once the walls are up.

**Comment by senhongo** [comment]
Plan was to assemble the remaining framing and stand them up yesterday and today. That didn't happen. I was distracted by work and also developed a slight fever yesterday so took the day off. Today was meant to catch up on yesterday's work.

I'll put it up over the weekend, and we can also start working on the roof as well.

**Comment by senhongo** [comment]
Today's progress.

Assembled and stood up the front wall and remaining half of back wall. Also added the centre roof rafter and hurricane ties for a clearer picture of what the roof will look like.

Tomorrow, we will complete the roof. Then, the remainder of the wall panels.

**Comment by senhongo** [comment]
Today's progress

Added roof rafters and hurricane ties. Took a while to decide how the overhang should be.

I've come to a decision that the overhang will be 40cm, maybe 35cm.

Next is to measure, cut and assemble the side edge rafters, then fill in the missing parts of the wall.

**Comment by senhongo** [comment]
*This was written yesterday but I forgot to click the submit button.*

After sleeping over it, I've come to a few conclusions.

I don't need an overhang on the north or east side.

That simplifies a few things. I don't have to be too any about the positioning of the rafters, because I'l be loping them off on both ends anyway.

- the south side will have an overhang of 30cm
- the overhanging edges on the north side will be cut off with a straight line draw against the wall
- I may want to unscrew all the rafters and move then 'forward', so there's less wastage
- no overhang on the east side means one less side rafter
- rather than do all pieces at once, start with 1 piece, cut it till the ideal length, then use it as a template for the rest

Maybe, instead of cutting the northern overhangs in-situ, I could place them, draw a line off the edge of the wall, then cut them with the circular saw. Probably faster overall.

**Comment by senhongo** [comment]
There has been a hiccup. Matsuzawa from Hakuba Glass came by to take a some measurements for the doors and windows, and found that the openings I created were too small. Basically, I looked at the wrong numbers on the spec sheet.

The options were to enlarge the openings by butchering the current framing or switch to a smaller model and adding more material. I opted for the latter option as it sounded less error prone for an amateur.

This means keeping the same width and only lowering the head piece for the door opening and adding a 2x4 to the bottom frame. I'll also have to lower the head piece for the window but can simply close off the window opening on the right (when looking from the inside).

**Comment by senhongo** [comment]
- [x] make template roof rafter
- [x] remove all current roof rafters
- [x] cut them according to template
- [x] re-frame the roof rafters
- [x] remove door head piece
- [x] add door floor piece
- [x] measure and cut jack studs in place
- [x] reassemble head piece
- [x] add supporting cripple for window piece
- [x] remove head piece (have to remove upper wall panels too)
- [x] measure and cut left jack stud in place
- [x] measure and cut head piece to size
- [x] add semi-jack stud
- [x] add double semi-king stud

**Comment by senhongo** [comment]
Plans for this weekend is to 

- [ ] remedy the window and door openings
- [ ] complete the roof rafters
- [ ] mount roof panels
- [ ] mount all remaining wall panels
- [ ] apply paper to outer walls

**Comment by senhongo** [comment]
Made a lot of progress today. Could've made more if I had woken up earlier as planned.

- rafter template made and rafters cut to size
- non-edge rafters re-framed
- I figured out how to mount the right side edge rafter as well as the studs for the triangular walls.
- door opening fixed
- window opening fixed
    - only need to cut the  left jack stud, re-frame the head piece and add the top supports.

Depending on the weather tomorrow, I should be able to finish the window openings and also complete the roof rafters.
Even if it rains, I can cut the roof rafters in my garage.

I need to compare the length of the rafters to the roof panels. If they're long enough, I don't have to trim them and could possible start mounting them already. I might still cut them in half anyway to make it easier to manipulate.

**Comment by senhongo** [comment]
Also, I need to go buy a longer ladder.

**Comment by senhongo** [comment]
I was feeling physically exhausted today and couldn't do much. I managed to finished the window opening.

I'll complete the rafters on tomorrow and start mounting the roof panels.

Some thoughts of the rafters. The ones on the right need to have a gap on the studs for the rafter to go through. The right rafter can go all the way to the edge, it's weight will be born by the offset studs.

The left rafter is made of 2 long vertical pieces joined by horizontal studs roughly 900mm in length. There will be blocking running down the middle of the  studs that lines up with the left most stud of the front/backwalls. The blocking will also be attached to the studs of the triangular section of the walls. This section does not require any offseting.

Before I do the roof panels, I should add the backing plate against the rafters. I haven't calculated the height of this piece yet. It should be:

`1 slanted rafter + 1 panel + 5c of styrofoam + furring strip + 1 panel`

I'll make it taller than needed and trim the excess later.

**Comment by senhongo** [comment]
Just read that a huge storm is passing by tomorrow. Depending on the severity, I may not be able to do any work. We shall see.

**Comment by senhongo** [comment]
### Today's progress

Storm came and went. Too wet to do any work on the standing structure. Might need to wait a few days for it to dry out.

Finished cutting the timber for side rafter studs and also pre-drilled the holes. Did not do the middle blocking piece. Wanted to wait till I've actually assembled the current pieces and measure off that.


Cut the bottom plates of both triangular wall sections and also pre-drilled holes to mount studs. Re-did the measurements for studs but not confident of numbers. Will measure again next time when weather permits.

Things still left to cut
- middle rafter blocking
-  triangular wall studs

I should be able to assemble them quite quickly. An hour or two, including the cutting of the above pieces. Once those are done, I can finally start mounting the roof panels.

**Comment by senhongo** [comment]
It's been very rainy and cloudy. I might need a couple days of sun for everything to properly dry. We'll see.

Also want to apply anti-rot coating to the standing frame. Better late than never.

**Comment by senhongo** [comment]
On Wednesday

- [ ] buy safety harness
- [ ] mounting material for walls/top roof layer
- [ ] roofing sheet

**Comment by senhongo** [comment]
I'm thinking now, if I pre-cut all the roof panels too, I could probably do the entire thing at once in a full day. Maybe two days.

I would like to be done by this weekend.

I will take Friday and Monday off if I want to.

**Comment by senhongo** [comment]
Went shopping for the safety harness and didn't find anything in Nafco or Caines.

Found the roofing sheets, they're 5k for a roll. It's cheap enough that I don't mind buying a roll to test it out.

Furring strips were also cheap-ish.

Initially, my plan was to look for strips that are 40mm, but I think I could work work 20mm strips. I'll have to double them up and criss cross them but that'll still give me the additional thickness that I need.

I wanted to get screws but forgot what length of screws I was getting. I think it might have  been 45mm ones.

I also need to the the long screws to use on the roof.

I might also possibly be taking delivery of the windows and doors from Hakuba Glass, as well as the roof insulation from Yokoyama this weekend.

**Comment by senhongo** [comment]
### Today's progress
- completed the triangular wall
- attached the right side rafter
- assembled the left side rafter

Aim for this weekend, complete framing and walls, wrap paper, install the windows and doors.


### Tomorrow's tasks

- [x] install left rafter
- [x] install panels for left wall
- [ ] install panels for left rafter
- [ ] install panels for right wall
- [ ] install edge panels
- [ ] install roof panel layer 1


I've been lazy, I should get around to adding the final scaffolding piece, and also start thinking about how to add height to it.

**Comment by senhongo** [comment]
### Today's progress

Assembled the left side rafter. Positioned the middle blocks incorrectly because I measured from the wrong corner. The measuring was done on Friday evening and I was rushing to get it done with only about 5 minutes of sunlight remaining. I should've waited till the next day. I only noticed the measurements were off after carrying the bloody thing up the ladders on to the roof.

I also noticed that the 2nd stud of the front wall was off by 2cm. At this point, I'm not sure if the rest of the studs and hence the rafters are in the right position. I'll have to double check them tomorrow.

Anyway, a lot of time was wasted redoing the left rafter piece.

I managed to put up most of the wall panels on the left wall.

**Comment by senhongo** [comment]
Change of plans.
It started raining sometime in the night and looks to continue all day long. Possibly on Monday too.

I want to give the timber and panels time to dry out so will not mount any panels.
In stead, I will pre-cut the panels for the remaining parts.

- [x] right wall
- [x] back top edge
- [x] left wall top right
- [x] front wall
- [x] roof

I also want to pre-cut the papers.

Now might also be a good time to work on the final scaffolding.

**Comment by senhongo** [comment]
### Problem

The studs for the front wall are off by 1 or 2 studs.
Not sure how that happened. It can be fixed, for the wall, because they're still straight.


But the rafters are now off centre.

### Solutions

1. dismount the hurricane ties and move them to the correct spot.
2. Add blocking so we can mount the panels horizontally instead of vertically.

The 2nd option is 'easier' in that we don't have to re-do stuff.

The 1st option is more correct, but involves more work.

### Found the problem

I flipped the top and bottom plates.

Confirmed using the older bottom place. The marks line up if I flip it around.

### Conclusion

It's too late to fix the studs, but I will fix the rafters so at least that is straight.
I might add some horizontal blocking below the top plate later.

**Comment by senhongo** [comment]
Also measured the wall panels for the first time. Don't know why I assumed it was 5mm thick, it's actually 10mm (or maybe 9.5mm).

Either that means the roof is  89mm + 10mm + 89mm + 10mm = 198mm thick.

The screws we got were 150mm long. It will do. The other options were 180 and 200. 150mm was cheaper, and I didn't like the idea of the screw penetrating the bottom of the rafter.

Anyway, let's fix the rafters then get back to measuring and cutting the panels.

**Comment by senhongo** [comment]
- [ ] right wall triangular panel (cut from scrapes)
- [ ] front wall panels, or at least the ones around the door opening
- [ ] tape between panel edges
- [ ] wrap paper around the walls
- [ ] roof panels, left overhang
- [ ] roof panels bottom the whole section
- [ ] install window

**Comment by senhongo** [comment]
Front entrance
- height 1910
- left panel width ~270
- right panel width ~170

Panel above entrance
- height 640
- width full

Roof rafter
- length 3170
- panel length 3030
- short 140
- 4 panels, need 560 spare
- edge rafter width is 335

**Comment by senhongo** [comment]
Tomorrow morning

- [ ] apply anti rot coating to current exposed surfaces
- [ ] mount back wall bottom panels
- [ ] wrap back wall with paper
- [ ] mount left wall remaining panels
- [ ] wrap with paper
- [ ] cut roof panels (8 sheets)
- [ ] cut front wall panels
- [ ] how many panels do I have left?

**Comment by senhongo** [comment]
Next actions

- [x] cut panels for door openings
    - left, right and bottom of door
- [x] mount front wall panels, leave right corner open
- [x] paper over front wall
- [x] install door and window

**Comment by senhongo** [comment]
### Today's progress
Only had about 30 minutes of work time today.
- cut the panel above the door openings
- mounted both upper panels as well as left panel

### Tomorrow
Assuming Hakuba Glass will install the glass panels on Friday, I will need to complete the front wall as well as install the window and doors. That side, I also need to start working on the roof. Especially if it's going to rain on Friday and Saturday. I need to have both layers up by then.

**Comment by senhongo** [comment]
### Today's progress

- completed the front wall
- wrapped almost all the walls in paper
    - top row for right wall remaining
- installed door and window
- installed left rafter roof panel

### Tomorrow's tasks

According to the weather forecast, Friday and Saturday are confirmed rain days. So we will want to get the roof up ASAP.
- [x] mount right wall top panels
    - 36, 46, 56
    - the 56 tall piece could be composed of 2 pieces of scrap maybe?
- [x] apply last piece of paper to top row and bottom exposed bits
- [x] mount back wall top edge panels (do this right before roof)
- [x] mount roof panels layer 1
- [x] mount insulation
- [x] mount roof panels layer 2
- [ ] apply roofing paper

Others
- [ ] tape bottom part of paper. maybe trim slightly too
- [x] for the back wall, mark where the studs are on the paper so we know where to screw the cladding backings

**Comment by senhongo** [comment]
Almost completed all the tasks. If I had woken up properly instead of sleeping in, we would likely be done today.

I probably need another 2 hours or so to complete it.

**Comment by senhongo** [comment]
We will get up early tomorrow morning, 7am and power through the remaining bits before the rain arrives.

There probably won't be time to do blocking and front fascia, so let's just mount the top panels but without screwing in the end bits. The important thing is to get the roof water proof before the rain arrives.

**Comment by senhongo** [comment]
### Today's tasks

- [x] roof panel end
- [x] fill remaining space with insulation 42cm
- [x] add missing battens 1m
- [x] add 2nd roof layer
- [x] trim sides
- [ ] cover edges with paper
- [ ] apply roofing sheet
- [ ] front fascia

**Comment by senhongo** [comment]
Could not complete tasks as it started raining around 2pm, which was my time limit anyway.

There're still a lot more minor tasks to do before the framing be can be considered complete. I want to make a list of them.


### Saturday
- [x] chore: dump the small, unusable 2x4 end bits into a transparent trash bag
- [x] measure floor thickness, (probably 150 over)　177
- [x] confirm how thick the cladding battens need to be (particularly the left and front wall) 56 from wall to edge of door trim
- [x] measure the gap below the door frame, figure out what can fit in there 2cm tall, 6cm deep
- [x] measure the size of the floor framing, and the gap between surface and 火打
    - 6cm between surface and top of joint
    - horizontal from right, 77, 78,
    - vertical from back, 80, 805, 815
- [x] measure how much insulation is needed
    - at least 15 pieces required
    - left wall
    - 415 x 2265
    - 343 x 965
    - 417 x "
    - 374 X "
    - 387 x 2265
    - 362 x "
    - back wall
    - 398 x 2265
    - 416 x "
    - 421 x "
    - 378 x "
    - 415 x "
    - 415 x "
    - 415 x "
    - 293 x "
    - right wall
    - 356
    - 424
    - 417
    - 414
    - 417
    - 418 x 2260
    - front wall
    - 377
    - 403
    - 77
    - 310 x 430
    - 410 x 430
    - 418 x 430
    - 204 x 430
    - 177
    - 415
    - 327 x 2620

### Sunday
- [x] top edge of 2nd roof layer
- [x] paper around roof edges
- [x] roof sheathing
- [x] front roof fascia
- [x] paper around right wall top and bottom exposed bits
- [x] paper around front wall top exposed bit

**Comment by senhongo** [comment]
## The floor

What do I do about it?

First, I need to thicken it. Raise it up.
Second, I want to get new panels, the current ones  are warped and mouldy.

### Option one
Get new 24mm boards
¥5,800 on [Komeri](https://www.komeri.com/shop/g/g420038/)

### Option two
Layer 2 pieces of 12mm boards together
¥1708 on [Komeri](https://www.komeri.com/shop/g/g1943997/)

Cheaper, but more work required.
twice the amount of surface needing anti-rot coating,
need to tightly bond both pieces together with glue and screws

We have money but we don't have time. Let's go with option 1.

## Underfloor insulation
Depends on the gap between surface and iron joint. But I wonder if it's possible to place styrofoam between the frame.

## Layering

- carpet
- 12mm layer panel
- 50mm layer insulation (x2?)
- 24mm layer panel
- plastic sheet

**Comment by senhongo** [comment]
## Definition of complete roof

- all corners sealed
- all panels applied
- has roofing sheet applied over it

**Comment by senhongo** [comment]
## Gap discovered

Discovered that there's a gap in the front where it meets the deck. The front panels are laid on top of the decking, but the studs and pushed up against the deck, so there's a small section where it's still exposed. We have to fill that.


## Stud top blocking

Also spent some time thinking about the blocking. The rear wall and front wall are straight forward. The right wall doesn't need any.

The left wall is the problem. My solution is to ignore it. I'll wrap the section in the thick plastic wrapping, seal the sides and fill it with foam.

## Floor conclusion

We have a 177 gap between the top of the framing surface to the bottom of the door trim, and then another 37 to flush of the door entrance. Considering the large gap, it's probably unnecessary to fill in the square gaps. So we'll simply lay a plastic sheet over the foundations, lay the 24mm panel on top of it, lay 100mm of insulation, and find another way to fill another 94mm.

Actually, one idea I have is to lay some 2x4s vertically on of the foundation, place the panel on top of that, then insulation, then a final piece of panel.

**Comment by senhongo** [comment]
## Today's progress

Completed the roof.
Wrapped and taped missing sections of the wall.

**Comment by senhongo** [comment]
Yokoyoma came with the roof guy to check.

Things we need to do this weekend

- [ ] on the right and back wall, add 2 pieces of 2x4 to give it the same height as the front and left overhangs.
- [ ] around the perimeter of the roof, add a 30x40 piece on the top edge
- [ ] around the perimeter of the roof, add a 30x40 on the bottom edge
- [ ] we should also strengthen and add another layer to the scaffolding so we can reach the top of the walls

**Comment by senhongo** [comment]
## Today's tasks

### Prep roof
- [x] measure lengths needed for 2x4
- [x] cut as necessary
- [ ] apply anti-rot coating

### Prep floor
- [x] measure floorboards
- [x] cut as necessary
- [ ] apply anti-rot coating

### Expand scaffolding
- [x] strengthen current layer
- [ ] add a 2nd, higher level

**Comment by senhongo** [comment]
## Today's progress

Measured and cut the 2x4 for the roof liner.
Didn't do anti-rot coating as I need to get 30x40 as well and would rather do it all at once.

Spent a bit of time on the scaffolding, mostly building on the east one and thinking about how to add a second layer.

I think I might not need scaffolding for the rear, but unsure.

I will be going to buy some 30x40 now.

## Tomorrow
- [x] apply anti-rot coating to all the things.
- [x] assemble the roof liner
- [ ] scaffolding layer 2

**Comment by senhongo** [comment]
## Today's progress

- coated the floor and roof liners with anti-rot
- fixed the floor panels, still a few screws more but battery ran out
- assembled the roof liners for the difficult parts, the back and right sides
- did not work on scaffolding

## Tomorrow's tasks
- [x] complete roof liner
- [x] complete floor panel screws

**Comment by senhongo** [comment]
## Today's progress

Did what we set out to do. Complete the roof liners and securing the floor.

## Tomorrow's tasks

I want to start sealing the building to run the heater in it, so start from the bottom and work out way up.

Maybe work on the scaffolding too.

**Comment by senhongo** [comment]
## Today's progress

Taped the insides. Was planning to use the wood panels on the bottom front walls, but eventually decided it probably wasn't necessary. Same for the top back wall.

Added blocking to top front wall.

Was planning to run the heater today but it wouldn't start. Will try to disassemble it tomorrow to see if it's possible to fix it.

## Tomorrow's task

Try to fix the heater. Once that is done, I want to start adding insulation to the walls, then wrap the internals. Need to figure out where the power lines and internet will come from. Most likely from the right wall.

Work on the scaffolding.

**Comment by senhongo** [comment]
## Today's progress

- added 2nd layer to right and left scaffolding
- additional brace to rear scaffold

Turned out better that expected. The makeshift ladder didn't work though. Will try unfolding the long ladder tomorrow to see if that can reach the top.

## Tomorrow's tasks

- [x] find a way to climb onto the 2nd layer
- [ ] caulk edge of floor panels
- [x] start adding insulation to walls
- [x] run heater against rear and right walls

## Next actions
- [ ] seal floor with plastic wrap to stop bugs coming in
- [ ] figure out the electricals

**Comment by senhongo** [comment]
## Today's progress

- tied a ladder to the right wall, confirmed that climbing up is possible
- same with left wall
- added some rails to the back wall that can be used as steps and to sit on
- added foam insulation to the front wall

Remaining actions, instead of working in steps, we'll work agilely.
- add a 50mm panel
- add a 30mm panel, once 1 wall section is complete, cover with plastic sheathing

## Next actions
- [ ] complete front wall big pieces insulation
- [ ] complete right wall big pieces insulation
- [ ] complete left wall big pieces insulation
- [ ] complete back wall big pieces insulation
- [ ] fill in the gaps with the smaller leftover pieces
- [ ] buy 50mm and 30mm insulation panels, as much as available
- [ ] caulk the gaps in floor panels
- [ ] lay plastic sheathing

**Comment by senhongo** [comment]
Above tasks moved to #11 as they're not framing related.

Framing tasks are DONE! This issue can finally be closed.
